index.html

```diff
@@ -2,59 +2,223 @@
+          <li><a href="https://twitter.com/"><img src="./images/twitter.png" alt="Twitter Icon"></a></li>
+          <li><a href="https://www.linkedin.com/in/"><img src="./images/linkedin.png" alt="Linkedin Icon"></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt="Medium Icon"></a></li>
+          <li><a href="https://github.com/"><img src="./images/github.png" alt="GitHub Icon"></a></li>
+          <li><a href="#"><img src="./images/Fingers icon.png" alt="Fingers Icon"></a></li>
+        </ul>
+      </div>
+  </header>
+
+  <section class="work-section">
+      <div class="achievement">
+      <div class="img">
+        <img src="./images/Desktop3.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Tonic</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Desktop-4.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Multi-Post Stories</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Snapshoot3.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Tonic</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Snapshoot4.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Multi-Post Stories</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
```

Comment:
- [ ] I think you can improve the responsiveness and look of the project images, they don't respond well on different mobile screens. 👍 
- - The first two project images overflow their container:
![image](https://user-images.githubusercontent.com/107632780/229259041-695982b5-5185-4c13-8549-aab7a21edc2e.png)


- - Also the images of the last two projects are too small and doesn't fit well with their container:
![image](https://user-images.githubusercontent.com/107632780/229259020-c8f720f3-9213-4435-ab3b-d9a2c8d71981.png)

- Please consider fixing this as your project is meant to look like the provided Figma design, and it will also make your portfolio look professional to the viewers. thanks 😄